pandas/core/reshape/concat.py

```diff
@@ -66,6 +66,9 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     Returns
     -------
     concatenated : object, type of objs
+        When concatenating all ``Series`` along the index, a ``Series`` is
+        returned. When ``objs`` includes at least one ``DataFrame`` or when
+        concatenating along the columns, a ``DataFrame`` is returned.
```

Comment:
I would maybe make the "concatenating along the columns" a separate case (so new sentence). Like "Concatenating along the columns (axis=1) always results in a DataFrame"